docs/integrating-scrapy.md

```diff
@@ -12,23 +12,23 @@ sidebar_label: Integrating Scrapy projects
 
 To run the migration tool, you need to have the Apify CLI installed. You can install it using Homebrew with the following command:
```

Comment:
Mmm, I dont like how we have two sets of install instructions in two separate files 😢 but I also dont wanna block it. If its easy, lets make this whole section just `Install cli by reading this`

Reply:
Fixed, thanks. I added prerequisity similar to quick start section.